### DISCUSSION TITLE
💪 VIP & Reputation System

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#4976] [Ideas]
The purpose of this discussion is to explore a new VIP and reputation system in Sunflower Land. With the launch of the new seasonal format, marketplace & free-to-play it is time to revisit and improve our current VIP system in the game.

The goal is to create an ecosystem that ensures a vibrant economy, supports free to players, mitigates botters/extractors & contributes to the project’s revenue.

**Problems**

1. Free-to-play introduces avenues for botters or extractors to earn ‘free’ resources, sell for SFL & impact economy prices
2. Real players are restricted to use the marketplace without VIP passes
3. The marketplace makes it easier for multi-accounters to funnel SFL & collectibles between each other.
4. The Seasonal Banner is less useful after the season race ends but the price is still expensive

# Reputation System

Instead of our current black and white VIP access system, we will introduce varying degrees of reputation that unlock different actions in the game. The goal is to give OG and real players access to more features whilst limiting botters or malicious actors.

Players can earn reputation points through a range of tasks:

- Reach spring island - 50 points
- Reach desert island - 50 points
- Reach volcano island - 100 points
- Connect Twitter Account - 100 points
- Connect Discord Account - 100 points
- Complete Proof of Humanity - 100 points
- Reach lvl 100 - 150 points
- Own a bud - 300 points
- VIP Access - 600 points

🔴 **VIP Access is different to reputation tiers.** 🔴 

VIP Access continues to give you ticket bonuses, discounts, VIP airdrops, XP bonus, Feed Bonus etc.

**Tier 1 (Beginners)**

- Maximum 5 trades total
- Cannot trade collectibles + wearables 🔒
- Cannot withdraw 🔒
- Cannot complete SFL deliveries 🔒
- Cannot participate in Auctions 🔒

**Tier 2 - 250 Points**

- Maximum 20 trades total
- 1 SFL delivery per day
- Can trade collectibles & wearables ✅
- Can collectibles & wearables ✅
- Can participate in Auctions ✅

**Tier 3 - 350 Points**

- 3 trades per day
- 2 SFL deliveries per day
- Can withdraw SFL ✅

**Tier 4 - 600 Points**

- Unlimited trades
- Unlock Goblin Exchange
- Unlock more SFL deliveries
- Can withdraw resources ✅

**Tier 5 - 800 Points**

- Unlock more SFL deliveries & bounties
- Open to more suggestions for top Tier players? 🤗 

### Reputation goals

In the above Tier system, we are aiming for the following goals:

Tier 2 - Players can reach with gameplay and social integrations. You can earn small amounts of SFL to use in game.
Tier 3 - Players have proven 'humanity' to a certain degree. Allow them to earn more SFL and more buy, swap, sell actions

# VIP Access

We are also planning a new system which gives players more flexibility around purchasing VIP access. Instead of buying a Seasonal Banner, players will purchase VIP access for a set period of time.

1 month - $4.99
3 months - $11.99
1 year - $39.99

Regardless of the time in the season, the prices will remain the same. For players joining mid-season, prices now become more accessible as they can consider a 1 month passes. The lifetime banner will continue to give holders unlimited **VIP access**. Going forward players can no longer buy it. Only players lucky enough to get it before the change will have lifetime free VIP

_Notes:_

- Players with lifetime banners will have unlimited VIP access (Lifetime banners are no longer an option to buy)
- Seasonal Banners will now be cosmetic items gained through the VIP chest.

### Considerations

1. One concern in the proposal is people sharing buds to access the Tiers. We are adding some more prevention around this 😏
2. We will need to notify players multiple times before their VIP access expires - this will prevent people completing seasonal tasks and expecting to get the bonus tickets

## Conclusion

Thanks for taking the time to read this proposal! As always, we would love your thoughts and feedback 🤗

**Comment by Xeko-92** [comment]
Edit: Sorry, didn't realize that VIP was different from the tiers. I now understand that you still need the VIP banner for bonus tickets and the VIP airdrops.

**Reply by adamhannigan** [reply]
Hi @Xeko-92 thanks for the feedback.

Can you please elaborate on the Lifetime Banner? 

We don't have any proposed changes to how it currently works.

**Reply by Xeko-92** [reply]
I'm curious as to what it even does then. My understanding is it would get me the equivalent of tier 4 privileges here, but I'd be getting that either way without the banner.

**Reply by adamhannigan** [reply]
There aren't any proposed additions for Lifetime Banners for the reputation system - it will be treated the same as a normal VIP access 👍

**Reply by Xeko-92** [reply]
I might be confused about the proposal. Will people still need to buy the banners to be able to get the bonus tickets during the season as well as the monthly items?

**Reply by adamhannigan** [reply]
@Xeko-92 Fair points - I've just updated the proposal to clarify this 👍

**Comment by heatedlime** [comment]
I’m guessing reputation rating can fluctuate/decrease since you can sell a bud and or discontinue VIP access?

**Reply by adamhannigan** [reply]
That is correct.

One concern in the proposal is people sharing buds to access the Tiers. We are adding some more prevention around this 😏

**Comment by Blasta-alasta** [comment]
So if you have 600 points without VIP, you don't need to buy VIP? Or will there be additional perks with VIP?

**Reply by adamhannigan** [reply]
@Blasta-alasta There has been some confusion around this on Discord as well. We have just made some updates to the proposal.

VIP Access is separate to reputation tiers.

VIP Access continues to give you ticket bonuses, discounts, VIP airdrops, XP bonus, Feed Bonus etc. 👍

**Reply by Blasta-alasta** [reply]
Ah cool. Yea i like this proposal then!

**Comment by KyawNaingThu** [comment]
Thank you for taking non-VIP players into consideration. I like this proposal.

**Comment by vitt0** [comment]
> Tier 2 - Players can reach with gameplay and social integrations. You can earn small amounts of SFL to use in game. 

I think this shouldn't be the case, for multiaccounts it's is pretty easily to just also do multiple socials account, so basically they aren't "being stop" by this reputation, just connect Discord, Twitter and reach Sprint Island and they already have enought reputation to extract.

I think it would be better to reduce greatly the points the socials give and also Sprind island, make it that to reach Tier 2 they need to either have go true the proof of humanity or reach Desert Island at least

**Reply by adamhannigan** [reply]
That is fair feedback 👍 

One outcome I'm aiming for Tier 2 is give new players the "illusion" of earning SFL - i.e. they get a taste for it but can't withdraw and limited by funnelling it through the marketplace

**Reply by vitt0** [reply]
If they can trade freely collectibles & wearables at the market not being able to withdraw SFL wouldn't be a issue for them, they can trade with their main account that went true the proof of humanity and withdraw from there, so I think it's important to reduce the simplicity for them to extract, using only socials to rely that they are actual players sounds like a low filter Imo

**Reply by adamhannigan** [reply]
@vitt0 That's good feedback 👍

**Comment by YotingoMTL** [comment]
I like the proposal. It would give people some options if they don't want to pay for V.I.P. or just can't. I am not sure tho if we need all these tiers. Maybe shortening it to 3-4 tiers and if you own the V.I.P. pass you instantly get max tier.

**Comment by Rudolph-the-Bumpkin** [comment]
Maybe it would more effective to block some specific actions inside the marketplace, such as:

1) Can't accept buying offers from other players: this would force players to respect the market price and a multi account wouldn't be able to transfer resources from multiple accounts to a core one. We didn't have this feature before so it should be ok to limit it. 


2) Withdrawal limits: Another reason to multi acc is to accumulate resources and withdraw. With no exit from accepting offers, they would need to grind until reaching a specific SFL volume, that could take enough time to make them quit. 

This way you don't necessarily block the free to player game loop, I guess. 

I would also give higher points to game achievements, such as higher levels and islands. I would also consider number of Farm Hands, since it would give another reason for players to build beds and create more demand for their resources - and give them reasons to shop new collectibles or power ups. Would be a nice way to introduce the farm hand feature. 

Regarding the Tiers, it might be interesting to create more than 5 levels to give players another thing to achieve / look for. Plus, granting amazing benefits to those in the end game, such as way more trading offers (both to list and to accept) and even lower withdrawal /trading fees, could incentivize players to reach Volcano Island, for example. 

Gameloop: 
1) Join the game
2) Progress in-game leveling up and expanding (core gameloop)
3) Reach new islands to increase your reputation
4) Earn the right to create selling orders
5) Increase the number of selling orders available when reaching higher reputation ranks
6) Earn the right to accept orders
7) Increase the number of buying orders you can accept per day
8) Increase the amount of SFL you can hoard per day (or maybe increased hoarding limits overall)
9) Reduced in-game trading fees (Top of the top, limited to real OGs and dedicated players - who are probably heavy users of marketplace)

What do you think of considering NFTs in wallet to also increase reputation? 
Someone who owns lots of collectibles (collectors) should have higher reputation than those who don't own any?

**Reply by adamhannigan** [reply]
Some great feedback 👏 

We definitely want to consider closing the extracting loops, until the player is confirmed 'real'.

I am a bit concerned around offering to many game achievement points - they can be a bit harder to prevent easy strategies and/or botting to attain.

I agree on adding some more tiers (as long as they are all simple to understand) ✅

**Reply by Rudolph-the-Bumpkin** [reply]
Definitely, simplicity is needed. 

I would love to see this reputation system having also a dynamic factor to it. Reputation system is something that not only makes sense to a crypto game but also to a *farm simulator*.

What would increase reputation game-wise in SFL metaverse?
- NPC Gifting
- NPC Deliveries
- Completed Bounties
- Top 100 faction weekly rank (?)
- Number of farm hands in your farm (more farmers = more reputation)
- Daily Chest streak (?)
- Treasure Island streak (?)
- Fishing Codex (?)
- Flower Codex (?)

It would be great if the reputation system actually introduce players to an expected behavior regarding specific features. We want players to engage into different features and it's easy to define specific goals for each farm with each feature. 

However, not every feature is user friendly to new players. Some will make them expend more money than they should and take their focus from expanding / leveling. 

I personally like a model where you have different tiers, related to different features that will change according to the player's progress (level and isle). 

I know it sounds complex, but it could be really simple ingame aslong as the player has one thing to focus at a time.

**Reply by adamhannigan** [reply]
Good ideas in there 👌 

The first one "NPC Gifting" makes a lot of sense. We already have a relationship system inside of our game, that uses deliveries, chores & flowers to keep track.

We could use your friendship scores as simple way to track (instead of lots of different mechanics)

**Reply by Rudolph-the-Bumpkin** [reply]
I like it!

And would love to see this friendship and relationship being a core part of the game! 

Friendship points would play as the "dynamic part" of the score?  Maybe working as a multiplier factor

**Reply by Rudolph-the-Bumpkin** [reply]
Another thing that could give reputation score is buys vs sells. If a player has a balance between both / buys more than sell, than he could have higher reputation.

**Reply by Rudolph-the-Bumpkin** [reply]
Buys vs sells would also show if the farm is profiting or not, which could also be a great indicator for players. Like, making the farm profitable could be the player's "main goal" inside the game. In order to do that, he must expand their farm and skills and collect SFTs to increase production and gains.

We could explore a "crypto wallet" UX inside the marketplace, like the ones we find in CEX apps. 

Could even show the assets the player is holding into a pizza chart, understanding how many SFL in different in-game tokens for example and each token dominance into their wallet (in this case, not a wallet, but maybe a shed?).

A little bit off topic, but just showing how this indicator (buys vs sells) could be explored not only for reputation reasons but to emphasyze Sunflower Land as a Farm Management game.

**Comment by Dionis404** [comment]
I would like players who are actively playing to have more reputation than those who have reached certain levels and performed certain actions.

**Comment by HinataShoyo5106** [comment]
More Points for reaching different islands should be really important to the reputation system.
The update on the Codex and the archievements shoudl also be updated and be added to the points like a milestone or smtng like that.
Factions participation should gain reputation points or boosts
Players with extremely imbalanced "gifts" like transferring to a single core account should trigger reputation reviews or penalties.

**Comment by aakifajmal** [comment]
The reputation - earn points UI displays 600 points for VIP access, but in the VIP purchase UI it shows 500 points. It that a miscalculation?

**Comment by Kernelgg** [comment]
GM! 

## What is this about
I'm hoping we can kinda improve in filtering bots using the REP system while motivating players especially new ones out there (and here).

Note. rn, max rep = 1,450
	  GH = 1k
## What's for vets
For serious players/vets especially those who **achieved max REP**, maybe we can give them some other benefits for their achievement.

## Suggestions
On the other hand, we can introduce ways to earn **Bytes** of rep to help dedicated players increase rep through fair effort and time.

I may suggest the following;

play for a certain amount of days or;
obtain a pet/nft pet or;
weekly task that gives rep or;
socializing with players;
content creator rep; and/or
limited time events/games participation.

We can also adjust quests rewards and stocks through rep (maybe a npc rep/relationship system that can increase and decrease depending on players' action).


## Bot filter
Projects like **Yakkamon** can bring waves of 'new' players into the world of sfl. I think REP would be a **great additional filter** for times like this. Adding activities designed for humans is a good way to help real players and discourage botting.
